src/setup.c

```diff
@@ -90,6 +99,26 @@ static void setup_gpio(void)
 	gpio_mode_setup(GPIOC, GPIO_MODE_AF, GPIO_PUPD_NONE,
 			GPIO6 | GPIO7 | GPIO8 | GPIO9);
 	gpio_set_af(GPIOC, GPIO_AF3, GPIO6 | GPIO7 | GPIO8 | GPIO9);
+
+	/* Bluetooth */
+	gpio_mode_setup(GPIOA, GPIO_MODE_AF, GPIO_PUPD_NONE, GPIO9 | GPIO10);
+	gpio_set_af(GPIOA, GPIO_AF7, GPIO9 | GPIO10);
+	gpio_set(GPIOA, GPIO10);
+}
+
+/**
+ * @brief Setup USART for bluetooth communication.
+ */
+static void setup_usart(void)
+{
+	usart_set_baudrate(USART1, 921600);
+	usart_set_databits(USART1, 8);
+	usart_set_stopbits(USART1, USART_STOPBITS_1);
+	usart_set_parity(USART1, USART_PARITY_NONE);
+	usart_set_flow_control(USART1, USART_FLOWCONTROL_NONE);
+	usart_set_mode(USART1, USART_MODE_TX_RX);
+
```

Comment:
why is not needed usart_enable_idle_line_interrupt anymore?

Reply:
That is part of the serial reception code (not implemented). Not sure I will implement it in this PR.

Reply:
:+1: